opentelemetry-sdk/src/opentelemetry/sdk/metrics/export/controller.py

```diff
@@ -61,3 +62,46 @@ def tick(self):
         detach(token)
         # Perform post-exporting logic
         self.meter.processor.finished_collection()
+
+
+class DebugController:
```

Comment:
I wonder if this should live in the test utils package rather than the SDK itself, any thoughts?

Reply:
yeah, good idea, moving it away